### PR TITLE
Create drop down menu to separate sky surveys from the rest of the images

### PR DESCRIPTION
**Pull request by ylvaselling**
Also fixes bug with sky survey images that the title didn't show in the selected images list

### CODE REVIEW

File: src/components/BottomBar/SkyBrowserPanel.jsx [hunk 1]
```diff
@@ -263,38 +265,74 @@ function SkyBrowserPanel({ }) {
...
+        const skySurveyImages = imageList.filter((img) => !img.hasCelestialCoords);
+        imageListComponent = (
+          <FilterList
+            className={styles.filterList}
+            height={currentImageListHeight}
+            searchText={`Search from ${skySurveyImages.length.toString()} images...`}
+          >
+            <FilterListData>
+              {skySurveyImages.map(item => {
+                return <SkyBrowserFocusEntry
```
Comment (WeirdRubberDuck) [inline src/components/BottomBar/SkyBrowserPanel.jsx:317]: See previous comment

**Comment by WeirdRubberDuck** [comment]
Also, did we always use to get these errors? Get a bunch of them every time I select an image 🤔 

image

**Comment by WeirdRubberDuck** [comment]
Manysky surveys also seem to not load correctly, like this one 
image


Sorry for putting all that in here, but it was just easier when doing the testing. Feel free to ignore and we can just create issues instead

**Comment by ylvaselling** [comment]
This PR has gotten some performance-enhancing updates. Now the loading time to choose All Images and Sky Surveys should be gone. 

As for the images that don't exist, I think they have always been there (unless WWT added them recently), but they were probably further down in the list so we didn't click them as often. The problem is probably that we aren't using WebGL 2 still and that we are using a legacy version of WWT that doesn't support FITS.

**Comment by alexanderbock** [comment]
The bright gray lines around the dropdown menu stick out quite a lot since we don't have any other structure element with that bright colors.  What do you think of reducing the brightness of that border just a bit?
Looks great and I love the speed up!

**Pull request merged**